OrbitClientGgp/main.cpp

```diff
@@ -19,27 +19,33 @@ ABSL_FLAG(uint32_t, capture_length, 10, "duration of capture in seconds");
 ABSL_FLAG(std::vector<std::string>, functions, {},
           "Comma-separated list of functions to hook to the capture");
 ABSL_FLAG(std::string, file_name, "", "File name used for saving the capture");
+ABSL_FLAG(std::string, debug_path, "",
```

Comment:
This is merely naming but I'd call this log_directory instead of debug_path and "log file" instead of "debug file".

Reply:
oh yes this makes a lot of sense, will rename it, thanks!

Reply:
Done, thanks!